docs/diagrams/UiClassDiagram.puml

```diff
@@ -28,33 +30,45 @@ Class HiddenLogic #FFFFFF
 Class HiddenOutside #FFFFFF
 HiddenOutside ..> Ui
 
-UiManager .left.|> Ui
+UiManager .right.|> Ui
 UiManager -down-> "1" MainWindow
-MainWindow *-down->  "1" CommandBox
-MainWindow *-down-> "1" ResultDisplay
-MainWindow *-down-> "1" PersonListPanel
-MainWindow *-down-> "1" StatusBarFooter
-MainWindow --> "0..1" HelpWindow
-
-PersonListPanel -down-> "*" PersonCard
-
-MainWindow -left-|> UiPart
+MainWindow *-right->  "1" CommandBox
+MainWindow *-right-> "1" ResultDisplay
+MainWindow *-right-> "0..1" PersonListPanel
+MainWindow *-right-> "0..1" ContactDetailsPanel
+MainWindow *-right-> "0..1" MeetingListPanel
+MainWindow *-right-> "0..1" ContactMeetingsPanel
+MainWindow *-right-> "1" StatusBarFooter
+MainWindow -right-> "0..1" HelpWindow
 
 ResultDisplay --|> UiPart
 CommandBox --|> UiPart
 PersonListPanel --|> UiPart
-PersonCard --|> UiPart
+ContactDetailsPanel --|> UiPart
+MeetingListPanel --|> UiPart
+ContactMeetingsPanel --|> UiPart
 StatusBarFooter --|> UiPart
 HelpWindow --|> UiPart
+MainWindow --|> UiPart
 
-PersonCard ..> Model
-UiManager -right-> Logic
+PersonListPanel ..> Model
+MeetingListPanel ..> Model
+ContactDetailsPanel ..> Model
+ContactMeetingsPanel ..> Model
+UiManager -left-> Logic
 MainWindow -left-> Logic
 
-PersonListPanel -[hidden]left- HelpWindow
-HelpWindow -[hidden]left- CommandBox
-CommandBox -[hidden]left- ResultDisplay
-ResultDisplay -[hidden]left- StatusBarFooter
+Logic -[hidden]right- UI
```

Comment:
@ckcherry23 @tonishka which diagram do you guys prefer? They are both a bit cluttered but I think some clutter is unavoidable because of how many classes are involved.